jest.config.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  coverageDirectory: './coverage/',
+  collectCoverage: true,
+  collectCoverageFrom: [
+    '<rootDir>/{components,packages}/*/src/**.[jt]s?(x)',
+    '<rootDir>/{components,packages}/*/src/**/*.[jt]s?(x)',
+    '!<rootDir>/{components,packages}/*/src/**/{stories,example,fixtures}.[jt]s?(x)',
+    '!<rootDir>/packages/storybook/**',
+    '!<rootDir>/packages/api-docs/**',
+    '!<rootDir>/packages/storybook-components/**',
+  ],
+  coverageThreshold: {
+    global: {
+      branches: 95,
+      functions: 95,
+      lines: 95,
+      statements: 95,
+    },
+  },
+  setupFilesAfterEnv: ['<rootDir>/scripts/setupTests.js'],
+  testMatch: [
+    '**/{components,packages}/*/src/**/?(*.)(spec|test).js?(x)',
```

Comment:
is this the only line that's changed in the config? Would be nice if your PR didn't move the config and change the config at the same time :-)

Reply:
approved if this line is the only config change :-)

Reply:
Lines 6, 12 - 19 have also been added